open_spiel/julia/test/games_api.jl

```diff
@@ -0,0 +1,148 @@
+@testset "Games" begin
+
+@testset "check registered names" begin
+    game_names = registered_names()
+
+    expected = Set([
```

Comment:
I'm not sure this is a good idea; it adds another place that has to be updated when anyone adds a game, but without adding a check that could signal meaningful failure.

Reply:
I agree. However, this test case is directly copied from the Python side. I'm not sure how to improve this test case further. @lanctot any idea?

Reply:
Sorry I missed this. I agree with James (and we should remove that test from the Python ones too).

Reply:
I just removed this test case here.